modules/oe_whitelabel_list_pages/oe_whitelabel_list_pages.module

```diff
@@ -26,3 +31,61 @@ function oe_whitelabel_list_pages_form_oe_list_pages_facets_form_alter(&$form, F
     $form['actions']['reset']['#attributes']['class'] = array_unique(array_merge($form['actions']['reset']['#attributes']['class'] ?? [], $reset_classes));
   }
 }
+
+/**
+ * Implements template_preprocess_node() for the List pages node type.
+ */
+function oe_whitelabel_list_pages_preprocess_node__oe_list_page(&$variables) {
```

Comment:
Perhaps introducing a field formatter would be a cleaner approach? WDYT @drishu ?